### ISSUE TITLE
ESLint results

### ISSUE THREAD

**Issue by pdehaan** [mozilla/universal-search-content#18]
### Setup:

``` sh
% npm i eslint babel-eslint -D
```

And then add the following script to the package.json file:

```
"lint": "eslint .",
```

**NOTE:** Ideally we'd use something like Grunt/Gulp to run tasks, but npm is good enough for now.
### Running:

``` sh
% npm run lint
```

---

Here's the results from a semi-quirky ESLint setup:

```
$ npm run lint

> universal-search-content@0.0.1 lint /Users/pdehaan/dev/github/universal-search-content
> eslint .

src/adapters/search_suggestions_adapter.js
  14:118  error  Missing semicolon  semi
  15:115  error  Missing semicolon  semi

src/models/activity_result.js
  24:3  error  Unexpected trailing comma  comma-dangle

src/views/index_view.js
   3:7   error  NavigationalSearchSuggestionsIndex is defined but never used  no-unused-vars
  53:49  error  Missing semicolon                                             semi
  55:28  error  Expected '===' and instead saw '=='                           eqeqeq
  74:47  error  Missing semicolon                                             semi

webpack.config.js
  2:6  error  webpack is defined but never used  no-unused-vars

✖ 8 problems (8 errors, 0 warnings)
```

---
### /.eslintrc

``` yaml
parser: babel-eslint

env:
  browser: true
  es6: true
  node: true

rules:
  indent: [2, 2]
  no-console: 0
  no-underscore-dangle: 0
  no-unused-vars: [2, {vars: all, args: none}]
  no-use-before-define: [2, nofunc]
  quotes: [2, single]
  strict: 0
```

---
### /.eslintignore

``` yaml
dist/
```

**Comment by pdehaan** [comment]
Updated ESLint config and output per @nchapman's recent Babel+Ampersand.js refactorizations.

**Comment by pdehaan** [comment]
FIxed in #20. Closing.
GO TEAM!
